docs/faq.md

```diff
@@ -116,3 +116,18 @@ a new benchmark.
 
 Please [file an issue on GitHub](https://github.com/google/fuzzbench/issues/new)
 or send a pull request fixing the problem.
+
+## How can I cite FuzzBench in my paper?
+
+You can use the following BibTeX entry:
+{% raw %}
+```
+@misc{fuzzbench,
+    author       = {Google},
```

Comment:
As discussed on chat, lets use an authors list here. You and @jonathanmetzman can work together on an ordered list of core contributors. No rush, whenever you get a chance.

Reply:
ping!

Reply:
Added two options: blog post or repo -- let me know what your preferences are.

Reply:
I thought we'd pick one of these options

Reply:
ah, lets do the github one. sorry shouldn't have committed in haste, can you fix this in master, or post a new pr.